packages/core/contracts/l2/polygon/lib/Merkle.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
```

Comment:
seeing this code was pulled from another repo, do you have any information on if it was audited in their repo? did you need to change the code at all?

Reply:
They are undergoing audit right now so this code might have to change...

I have only made changes in RLPReader and I've added comments indicating that.

Reply:
I'll also add comments for contracts that I made no changes to.